src/goto-instrument/dump_c.cpp

```diff
@@ -894,7 +894,9 @@ void dump_ct::convert_global_variable(
   }
 
   if(!func.empty() && !symbol.is_extern)
-    local_static_decls[symbol.name]=d;
+  {
+    local_static_decls.insert(std::pair<irep_idt, code_declt>(symbol.name, d));
```

Comment:
I _think_ that's fine, but I am surprised to see this seemingly unrelated change in here. And then `.emplace` would seem better?!

Reply:
It's related as follows: the PR enforces that a code_declt always has a symbol_typet parameter, which is not the case for a very brief time when using [] to insert an element into a map.
But will use .emplace.